src/molecules/dialog/index.tsx

```diff
@@ -10,16 +10,17 @@ import {
   DialogTitle,
 } from '@/components/ui/dialog';
 import AutoForm, { AutoFormSubmit } from '../../organisms/auto-form';
+import { ScrollArea } from '@/components/ui/scroll-area';
 
-export type tableAction = {
+export type TableAction = {
```

Comment:
If we are going to change the type name, can we check where did we use the type in the core project as well and update it there?